portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase

```diff
@@ -611,10 +603,8 @@ definition {
 			ACIndividualsDashboard.assertIndividualAttribute(individualAttribute = "New Segment Test");
 		}
 
-		task ("Delete the CSV Data Source") {
-			ACSettings.deleteDataSource(
-				dataSourceName = "CSV Job Title",
-				dataSourceType = "CSV");
```

Comment:
This is a complicated test, because it involves checking that segment membership updates when only *some* of an individual's attributes are removed. Specifically those attributes which gave them segment membership in the first place.

How we test this now:
1. Combine individual attributes from different data sources into one individual, like so:
| Data Source 1 | Individual 1 | Attribute A |
| Data Source 2 | Individual 1 | Attribute B |
2. Create a segment with criteria based on only one attribute (Attribute B)
3. Delete Data Source 2 where the individual got Attribute B from
4. Check to see if Individual 1 is no longer in the segment, because their Attribute B was deleted

So if I understand the change to the test, it follows these steps:

1. Add a data source with a DXP user
2. Add a segment with the criteria "email = ac@liferay.com"
3. Go to the individual and assert that segments tab shows that they belong to the segment you created
4. Delete the data source that the individual is from
5. Go to individual's segment tab and assert that they are no longer associated with the segment.

Does this actually work? I would expect that when you do step 4, it would delete the user completely, meaning we could no longer check their membership?
Or maybe I am missing something.

Reply:
Yes, you're right. The individual will be excluded from the property. This verification after the deletion is necessary to navigate inside the created segment and verify that in the membership list there is no data

Reply:
You are right Christian, thank you. I'm sending the changes :smile: